google/cmd/template.go

```diff
@@ -103,11 +104,21 @@ type Function struct {
 
 	// Region is used to determine whether the resource is dedicated to a region or not
 	Region bool
+
+	// API is used to determine the
+	// google API to use as defined in the Reader
+	// for a complete list of API: https://godoc.org/google.golang.org/api
+	// ex: compute, storage
+	// default goes to `compute`
+	API string
```

Comment:
I would potentially name it `Service`, they refer to "them" as `Google Cloud Services`.

Reply:
In Google SDK, `API` refers to `Google Cloud Service` and `Service` refers to a component of this API.

Example: Google Compute Engine is a `Google Cloud Service`, `compute` is the name of the API in the SDK and `InstancesService` is a `service` to manage `instances`. 

Replace `API` with `Service` could lead to potential errors, since we already have a `ServiceName` fields: https://github.com/cycloidio/terracognita/blob/7d4f144252d5e39ac230f6daa2f643a370d68c24/google/cmd/template.go#L99-L102